docs/howtos/cert-manager.md

```diff
@@ -12,16 +12,19 @@ Note: Ambassador Edge Stack will automatically create and renew TLS certificates
 
 ## Install Cert-Manager
 
-There are many different ways to [install cert-manager](https://docs.cert-manager.io/en/latest/getting-started/install.html). For simplicity, we will use Helm.
+There are many different ways to [install cert-manager](https://docs.cert-manager.io/en/latest/getting-started/install.html). For simplicity, we will use Helm v3.
 
-1. Install cert-manager
+1. Add the `jetstack` repository.
+    ```bash
+    helm repo add jetstack https://charts.jetstack.io && helm repo update
+    ```
 
-```
-kubectl create ns cert-manager
-kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v0.11.1/cert-manager-no-webhook.yaml
-```
+2. Install cert-manager
 
-**Note:** The resource validation webhook is not mandatory, but if implemented, requires additional configuration.
+    ```
+    kubectl create ns cert-manager
+    helm install cert-manager --namespace cert-manager --set installCRDs=true
```

Comment:
I very much disagree with the way jetstack chose to support installing CRDs in the chart. This approach has no safeguards against `helm uninstall` completely removing all of the CRs in the cluster. 

We should use their recommended way of installing the crds by installing them first with a `kubectl apply` and then using the chart to install the rest of cert-manager

Reply:
My only concern with this is that it results in versioning.  The URL path that they use includes the version number, i.e. https://github.com/jetstack/cert-manager/releases/download/**v0.15.1**/cert-manager.crds.yaml (emphasis mine).  Additionally, applying it requires you use `--validate=false` which I'm not so sure about either.

Reply:
You can use this: `https://github.com/jetstack/cert-manager/releases/latest/download/cert-manager.crds.yaml`

Reply:
Aha!  That's what I needed, I tried replacing the version number w/ latest and got 404'd. TYVM.